src/Home/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './Home.js';
```

Comment:
We should have 1 file per component from where we export the component. You can either put all your code in Home/index.js or use Home/Home.js. Usually Home/index.js is considered a standard practice.